doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -76,7 +76,7 @@ detailed instructions on forking. In short, if your GitHub username is
 https://github.com/mygithubaccount/ceph. Once you have created your fork,
 you clone it by doing:
 
-.. code::
+.. prompt:: bash $
 
     $ git clone https://github.com/mygithubaccount/ceph
```

Comment:
no longer need the $ here and a few other places below - it shows up doubled

Reply:
@jdurgin fixed and repushed.